utilities_common/bgp_util.py

```diff
@@ -208,16 +209,17 @@ def run_bgp_command(vtysh_cmd, bgp_namespace=multi_asic.DEFAULT_NAMESPACE, vtysh
         output, ret = clicommon.run_command(cmd, return_cmd=True)
         if ret != 0:
             click.echo(output.rstrip('\n'))
-            sys.exit(ret)
+            if exit_on_fail:
```

Comment:
>if [](http://example.com/codeflow?start=12&length=2)

Suggest to add an `else` block and assign empty string to `output`. Same as in `except` block.